.travis.yml

```diff
@@ -1,8 +1,24 @@
 language: node_js
 node_js:
   - "8"
-
 script:
   - yarn run lint
-
+  - yarn run generate-example-images
+  - yarn run build
 cache: yarn
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable (Over to you, dom)
+  keep-history: true
+  local-dir: dist
+  name: Dominik Moritz
+  email: domoritz@gmail.com
+  project-name: vega-editor
```

Comment:
Isn't the name vega/editor or just editor?

Reply:
Let's make it by the repo name i.e. `editor`. (Although not necessary)

Reply:
Can you just delete it? It's not really necessary.